backend/gn_module_monitoring/command/utils.py

```diff
@@ -116,6 +116,137 @@ def process_available_permissions(module_code, session):
         )
 
 
+def clean_existing_permissions(module_code, session):
+    try:
+        module = session.scalars(select(TModules).where(TModules.module_code == module_code)).one()
+    except NoResultFound:
+        print(f"Le module {module_code} n'est pas présent")
+        return
+    print(f" Reprise des permissions du module {module_code}")
+    # ########
+    # Mise à jour des permissions existantes vers les sous objets
+    # Création des permissions des sous-objets à partir des permissions contenus dans l'objet ALL
+    print(
+        " Création des permissions des sous-objets à partir des permissions contenus dans l'objet ALL"
+    )
+    session.execute(
```

Comment:
It would be beneficial to add error handling for the SQL queries in the `clean_existing_permissions` function. This would help in identifying and resolving any issues that might occur during the execution of the queries. [important]